reservations/urls.py

```diff
@@ -0,0 +1,7 @@
+from django.urls            import path
+
+from reservations.views     import ResevationsView
+
+urlpatterns = [
+    path('/reservation', ResevationsView.as_view())
```

Comment:
```suggestion
    path('', ResevationsView.as_view())
```
```python
GET :8000/reservations # 예약 리스트
POST :8000/reservations # 예약하기
```

현재 코드로는 `:8000/reservations/reservation` 일 것 같은데 suggestion과 같이 수정해주세요!